systest/cluster/cluster.go

```diff
@@ -86,10 +107,71 @@ type Cluster struct {
 	poets     []string
 }
 
+func (c *Cluster) persist(ctx *testcontext.Context) error {
+	if err := c.accounts.Persist(ctx); err != nil {
+		return err
+	}
+	return c.persistFlags(ctx)
```

Comment:
how about the config file? say if i were testing a change to config/config.go or config/preset?

Reply:
configuration is provided using flags. what about config file?